packages/replay-next/src/utils/suspense.ts

```diff
@@ -98,13 +102,36 @@ export function createWakeable<T>(debugLabel: string): Wakeable<T> {
   return wakeable;
 }
 
-type AnyFunction = (...args: any[]) => any;
+export async function convertSuspenseToAsyncAwait<ReturnType>(
```

Comment:
:hand: this looks like it's doing a superset of what I'd added in `createFetchAsyncFromFetchSuspense` at the bottom of this file :)  Should we replace all references to `createFetchAsyncFromFetchSuspense` with this function instead?

Reply:
Ah, good point. I thought there was a helper like this– and I searched for it! Guess I used the wrong search keywords.

Sure!